coding-guides/starboard.md

```diff
@@ -0,0 +1,154 @@
+# Starboard
+
+This is a long awaited feature, requested by many people.
+
+Let's begin by talking about what a starboard is. This is an example taken from the Discord.js Official Server.
```

Comment:
You show an image, that's it? where's the explanation? (I missed this the first time, my bad)